server/api.js

```diff
@@ -134,4 +135,15 @@ router.post("/publish-tenders", (req, res) => {
 	res.status(200).json({ message: "Form submitted successfully!" });
 });
 
+router.get("/buyerTender/:id", async (req, res) => {
```

Comment:
In the future, it would be good to separate the backend and frontend PRs. Usually you can build the backend without the backend and test it using curl, postman, or something similar.